src/open_inwoner/openzaak/tests/test_case_detail.py

```diff
@@ -1175,53 +1175,6 @@ def test_document_ordering_by_name(self, m):
         self.assertEqual(documents[1].name, self.informatie_object["titel"])
         self.assertEqual(documents[2].name, self.informatie_object_no_date["titel"])
 
-    @patch(
-        "open_inwoner.openzaak.clients.ZakenClient.fetch_status_history", autospec=True
-    )
-    def test_e_suite_missing_current_status_fetch_status(
```

Comment:
@alextreme I'm now unsure what is still and issue and what is not, but from what I understood from the taiga issue, this regression test is no longer needed?

Reply:
Indeed it can be removed